ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/SketchOnMap/README.md

```diff
@@ -0,0 +1,38 @@
+# Sketch on map
+
+This sample demonstrates how to use the Sketch Editor to edit or sketch a new point, line, or polygon geometry on to a map.
+
+![](screenshot.png)
+
+## Use case
+
+A field worker could annotate features of interest on a map (via the GUI) such as location of dwellings (marked as points), geological features (polylines), or areas of glaciation (polygons).
+
+## How to use the sample
+
+Choose which geometry type to sketch from one of the available buttons. Choose from points, multipoints, polylines, and polygons.
+
+Add points or verticies by tapping on the map. You can edit individual points by tapping to select the point, then dragging it to a new location or double tapping to delete it.
```

Comment:
Also, double tapping to delete will be removed. It seemed too unintuitive. I sent you some code for adding a delete custom context menu option. You could show this on right click for desktops or long press and hold (or something similar on mobile).